osf/models/registrations.py

```diff
@@ -988,11 +989,18 @@ def update_registration_responses(self, registration_responses):
         validated before this method is called.  If writing to registration_responses
         field, persist the expanded version of this to Draft.registration_metadata.
         """
+        registration_responses = self.escape_registration_responses(registration_responses)
         self.registration_responses.update(registration_responses)
         registration_metadata = self.expand_registration_responses()
         self.registration_metadata = registration_metadata
         return
 
+    def escape_registration_responses(self, registration_responses):
```

Comment:
Should this be `unescape_registration_responses`? Maybe `unescape_registration_file_names`?